generate_books.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const faker = require('faker');
+
+class Book {
+  generateRandomBook(bookId) {
+    const genre = faker.random.arrayElement([
+      'Fiction',
+      'Mystery',
+      'Science Fiction',
+      'Fantasy',
+      'Romance',
+      'Thriller',
+    ]);
+    const price = faker.finance.amount(5, 30, 2);
+    this.bookId = bookId;
+    this.genre = genre;
+    this.price = price;
```

Comment:
Instead of defining a price variable you can directly used as follows - 
`this.price = faker.finance.amount(5, 30, 2);`